app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client.rb

```diff
@@ -7,4 +7,8 @@ def initialize(ext_management_system)
 
     hawkular_client
   end
+
+  def system_metrics
+    @client.http_get('/metrics')
```

Comment:
@simon3z 
a. changed validity checking to use `availability` , it answers with much more compact struct, usually empty.
b. remove the unused `status` and `system_metrics` functions